source/main.cpp

```diff
@@ -31,6 +31,8 @@
 
 #define s_VERSION "0.2.0"
 
+int g_ERROR { 0 }; // zero initialized
```

Comment:
Don't need to comment language syntax. Its obvious from expression that its zero initialized.